packages/trader/src/Modules/Contract/Components/InfoBox/info-box-longcode.jsx

```diff
@@ -27,7 +28,7 @@ const InfoBoxLongcode = ({ contract_info }) => {
                 </Text>
                 {` `}
                 {contract_info.longcode.length > max_longcode_length && (
-                    <Text as='a' href='#' size='xs' onClick={handleToggle} c>
```

Comment:
@akmal-deriv removing `c` prop introduced in [this PR](https://github.com/binary-com/deriv-app/pull/8997/files#diff-39ab3d51fba0f51de820f3fdb9b49e967997cde04cc4c496d135f88e5cbbb90d) because it looks like a typo since there is no `c` prop in this component.